addons/collapse-footer/addon.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Auto-hiding footer",
+  "description": "Collapses the website footer and expands while hovering over it. Prevously part of \"Infinite scolling\".",
```

Comment:
This name and description would only make sense if the footer was always visible at the bottom of the screen without the addon.